core/src/test/java/cocktailofferer/service/image/command/ImageMetadataCommandServiceTestConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package cocktailofferer.service.image.command;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.ComponentScan;
+
+@ComponentScan
+@TestConfiguration
+public class ImageMetadataCommandServiceTestConfiguration {
```

Comment:
Having a config inside test class is less error-prone

Reply:
Why do u think so?

Reply:
Because we won't have to link test with its config via annotation parameter. Linking will be made automatically. So, we won't make linking errors when copying-pasting or refactoring test classes and their configurations. Also, it's easier to read a single file.

Reply:
I'm not sure about "easier to read a single file". Now configuration is small. But in the future it will might be bigger. It's hard to read big files

Reply:
Let's try the approach of placing simple configurations into test classes. It's easier to read and understand.